tailwind.config.ts

```diff
@@ -165,15 +165,15 @@ export default {
 				},
 
 				".heading-1": {
-					"@apply font-obviously text-7xl leading-tight": {},
```

Comment:
Looks like these styles aren't used in the Hero at all, it uses the `text-` utilities directly

All good if this was an intentional change as well, but it will scale down titles on all other pages slightly 👍

Reply:
Because the hero is smaller, heading-3 needs to be smaller to balance that out, and then because heading-3 needs to be smaller all of these need to be re-balanced as well. There might be a way around it but after testing this on every page and a bunch of blog posts I think it looks good / is safe to ship! I kind of prefer this stylistically, as well.